draft-ietf-scitt-architecture.md

```diff
@@ -366,287 +364,151 @@ Failure to produce this proof can indicate that the Transparency Services operat
-This attestation evidence can be supplemented with Receipts for the software and configuration of the service, as measured in its attestation report.
-
-### Configuration
+An important function of a Transparency Service is to maintain Registration Policies for the Append-only Log.
+The Append-only Log is the verifiable data structure which registers Signed Statements and supports the production of Receipts.
 
-The Transparency Service records its configuration in the Append-Only Log using Transparent Statements with distinguished media type `application/scitt-configuration`.
+All Transparency Services MUST expose APIs for the registration of Signed Statements and issuance of Receipts.
 
-The registration policy for statements with the media type suffix (`+<format>` is implementation-specific.
-The implementation SHOULD document them, for example defining the Issuers authorized to register configuration Signed Statements.
+Transparency Services MAY support additional APIs for auditing, for instance, to query the history of Signed Statements.
 
-The Transparency Service is configured by the last Transparent Statement of this type.
-The Transparency Service MUST register a Signed Statement that defines its initial configuration before registering any other Signed Statement.
-The Transparency Service MAY register an additional Signed Statement that updates its configuration.
+Typically a Transparency Services has a single Issuer identity which is present in the `iss` claim of Receipts for that service.
 
-The Transparency Service provides an endpoint that returns the Transparent Statement that describes its current configuration.
+Multi-tenant support can be enabled through the use of identifiers in the `iss` claim, for example, `ts.example` may have a distinct Issuer identity for each sub domain, such as `customer1.ts.example` and `customer2.ts.example`.
 
-The configuration `reg_info` SHOULD include a secure version number and a timestamp.
+### Initialization {#ts-initialization}
 
-The sample configuration payload uses the CDDL {{-CDDL}}
-
-~~~ cddl
-Signature_Algorithms = [ int ]
-
-Registration_Policy = {
-  * tstr => any
-}
-
-SCITT_Configuration = [
-  supported_signature_algs: Signature_Algorithms ; supported algorithms for signing Statement
-  ledger_alg: int ; type of verifiable data structure
-  service_uri: tstr; base URI of the transparency service, will be the issuer in the receipt CWT claim set
-  root_certificates: [ COSE_X509 ]
-  supported_apis: [ SCITT_Endpoint ]
-  registration_policies : Registration_Policy
-]
-~~~
+The Append-only Log is empty when the Transparency Service is initialized.
+The first entry that is added to the Append-only Log MUST be a Signed Statement including key material.
```

Comment:
The restriction on the first entry doesn't make sense to me. Why are we being so specific about the log contents ?